### PR TITLE
models added and migrations completed

### PR DESCRIPTION
**Pull request by chrlm**
titre clair

### CODE REVIEW

File: app/models/user.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+class User < ActiveRecord::Base
+  has_many :bookings, dependent: :destroy
+  has_many :places, through: :bookings
```
Comment (floangles) [inline app/models/user.rb]: `has_many :places`

File: app/models/place.rb [hunk 2]
```diff
@@ -0,0 +1,15 @@
+class Place < ActiveRecord::Base
+  has_many :bookings
+
+  validates :name, presence: true
+  validates :address, presence: true
+  validates :zipcode, presence: true
+  validates :city, presence: true
+  validates :description, presence: true
+  validates :name, presence: true
```
Comment (floangles) [inline app/models/place.rb]: Delete this line

File: app/models/place.rb [hunk 3]
```diff
@@ -0,0 +1,15 @@
...
+  validates :capacity, presence: true
+  validates :rooms, presence: true
+  validates :beds, presence: true
+  validates :rate, presence: true
+  validates :available, presence: true
```
Comment (floangles) [inline app/models/place.rb]: C'est du grand n'importe quoi !

File: app/models/user.rb [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  validates :first_name, uniqueness: { scope: :last_name }
+  validates :first_name, presence: true
+  validates :last_name, presence: true
+  validates :email, presence: true, uniqueness: true
+  validates :phone_number, presence: true
+  validates :address, presence: true
+  validates :zipcode, presence: true
+  validates :city, presence: true
+  validates :sex, presence: true
+  validates :description, presence: true, uniqueness: true
```
Comment (floangles) [inline app/models/user.rb]: uniqueness ?

File: db/migrate/20150622144407_create_users.rb [hunk 5]
```diff
@@ -0,0 +1,18 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
+      t.string :first_name
+      t.string :last_name
+      t.string :email
+      t.string :phone_number
+      t.text :address
```
Comment (floangles) [inline db/migrate/20150622144407_create_users.rb]: i would have right string instead of text

File: db/migrate/20150622144923_create_places.rb [hunk 6]
```diff
@@ -0,0 +1,19 @@
+class CreatePlaces < ActiveRecord::Migration
+  def change
+    create_table :places do |t|
+      t.string :name
+      t.text :address
```
Comment (floangles) [inline db/migrate/20150622144923_create_places.rb]: idem should be string

File: db/migrate/20150622145518_create_bookings.rb [hunk 7]
```diff
@@ -0,0 +1,16 @@
+class CreateBookings < ActiveRecord::Migration
+  def change
+    create_table :bookings do |t|
+      t.integer :guests
```
Comment (floangles) [inline db/migrate/20150622145518_create_bookings.rb]: hu?

File: db/migrate/20150622144407_create_users.rb [hunk 8]
```diff
@@ -0,0 +1,18 @@
+class CreateUsers < ActiveRecord::Migration
+  def change
+    create_table :users do |t|
```
Comment (floangles) [inline db/migrate/20150622144407_create_users.rb]: nope. users table already exists

**Comment by floangles** [comment]
Review done

**Comment by chrlm** [comment]
Fixes done
Models and migrations corrected

**Comment by floangles** [comment]
Yeah :+1:

**Comment by floangles** [comment]
:smiley_cat:

**Pull request merged**